docs/docs/develop/advanced-concepts/02-context.md

```diff
@@ -28,12 +28,12 @@ https://github.com/cosmos/cosmos-sdk/blob/v0.50.0-alpha.0/types/context.go#L41-L
 * **Header:** The [header](https://docs.cometbft.com/v0.37/spec/core/data_structures#header) is a Blockchain type. It carries important information about the state of the blockchain, such as block height and proposer of the current block.
 * **Header Hash:** The current block header hash, obtained during `abci.FinalizeBlock`.
 * **Chain ID:** The unique identification number of the blockchain a block pertains to.
-* **Transaction Bytes:** The `[]byte` representation of a transaction being processed using the context. Every transaction is processed by various parts of the Cosmos SDK and consensus engine (e.g. CometBFT) throughout its [lifecycle](../basics/01-tx-lifecycle.md), some of which do not have any understanding of transaction types. Thus, transactions are marshaled into the generic `[]byte` type using some kind of [encoding format](./05-encoding.md) such as [Amino](./05-encoding.md).
+* **Transaction Bytes:** The `[]byte` representation of a transaction being processed using the context. Every transaction is processed by various parts of the Cosmos SDK and consensus engine (e.g. CometBFT) throughout its [lifecycle](../high-level-concepts/01-tx-lifecycle.md), some of which do not have any understanding of transaction types. Thus, transactions are marshaled into the generic `[]byte` type using some kind of [encoding format](./05-encoding.md) such as [Amino](./05-encoding.md).
```

Comment:
how do you feel about keeping with the term basics? high level concepts may not be a term everyone knows

Reply:
hmm I found the previous approach a bit unclear, and there was a consensus within the team to transition away from it a while ago. This change  was aimed to enhance clarity but also for consistent user experience. We can revisit this with the team if u like though.

Reply:
high level concepts may be harder to understand with people that dont speak english was my thinking while reading this. Basic is something everyone knows